### PR TITLE
Hb 01

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,5 +8,68 @@
...
+        <jdk.version>11</jdk.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
+        <maven.checkstyle.plugin.configLocation>
+            https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
+        </maven.checkstyle.plugin.configLocation>
+    </properties>
 
-</project>
```
Comment (Nokafan) [inline pom.xml:12]: Empty line?

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert movie entity", e);
```
Comment (Nokafan) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]: Not sure, but maybe your custom exception is better here?

File: src/main/java/com/dev/cinema/util/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Error creating Session Factory");
```
Comment (Nokafan) [inline src/main/java/com/dev/cinema/util/HibernateUtil.java:16]: Same here, maybe custom?

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">admin</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="com.dev.cinema.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (Nokafan) [inline src/main/resources/hibernate.cfg.xml]: Empty line?

**Review by Nokafan — COMMENTED** [review]
Good job :)

File: src/main/java/com/dev/cinema/util/HibernateUtil.java (hunk 3 above)
Comment (Rudiachenko) [reply src/main/java/com/dev/cinema/util/HibernateUtil.java:16]: this exception has a different nature, so we should to throw RuntimeException here

Comment (Nokafan) [reply src/main/java/com/dev/cinema/util/HibernateUtil.java:16]: Agree. you are right.

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

**Pull request merged**